Framework/Core/README.md

```diff
@@ -182,12 +183,21 @@ Data flow parallelism is simply expressed by tuning the data flow, adding explic
 
     DataProcessorSpec{
       "tpc_processor_1",
-      {InputSpec{"TPC", "CLUSTERS", SubSpec(1)}},
+      Inputs{},
+      Outputs{{"TPC", "CLUSTERS", SubSpec(1)}},
```

Comment:
Yes, this is actually just an example. The for parallel takes a lambda so you can really modify the outputs as you prefer for each worker. This could be extended to support ranges as well (needs a bit of refactoring of the actual device topology construction, but should not be too difficult).